stdlib/public/stubs/Stubs.cpp

```diff
@@ -44,6 +46,9 @@
 #if defined(__CYGWIN__) || defined(__HAIKU__)
 #include <sstream>
 #endif
+
+#if SWIFT_STDLIB_HAS_LOCALE
+#include <clocale>
 #if defined(__OpenBSD__) || defined(__ANDROID__) || defined(__linux__) || defined(__wasi__) || defined(_WIN32)
 #include <locale.h>
```

Comment:
Why do we need this header?  If we are including `clocale` we should have all the same declarations already.